packages/flutter/lib/src/material/app.dart

```diff
@@ -266,7 +266,7 @@ class MaterialApp extends StatefulWidget {
     this.routerConfig,
     this.backButtonDispatcher,
     this.builder,
-    this.title = '',
+    this.title,
```

Comment:
What does this change in default values mean for Android (the only other platform besides web that uses this)?

> On Android the titles appear above the task manager's app snapshots which are displayed when the user presses the "recent apps" button

Does changing the defaults here changes what's displayed there? If so, we may have to leave the defaults in place and instruct people to pass in null explicitly if they don't want to set a title.

Reply:
@reidbaker can this cause any troubles for Android?

Reply:
Discussed offline with @reidbaker . We're not sure of the implications on Android. Changed the PR to make the behavior changes web only.